soh/soh/OTRGlobals.cpp

```diff
@@ -1290,3 +1290,57 @@ extern "C" int Controller_ShouldRumble(size_t i) {
 
     return 0;
 }
+
+extern "C" int GetSkulltulaCountMessage(char* buffer, char* src, const int maxBufferSize) {
```

Comment:
Why the return? It looks like it always returns zero and its never used anyway.

Reply:
This was a mistake and the return was supposed to be used by the caller of this function. Instead a strlen was added from some debugging that was done and was not replaced.